### ISSUE TITLE
[MU4 Issue] [Accessibility] VoiceOver doesn't speak dropdown values as they change

### ISSUE THREAD

**Issue by shoogle** [musescore/MuseScore#12974]
**Describe the bug**
When a dropdown box is expanded and you select a value using the arrow keys, VoiceOver should say the current value as it changes. Instead it just says "text" for each value.

**To Reproduce**
1. Launch VoiceOver (Cmd+F5).
2. Open the New Score dialog > Choose Instruments page.
3. Navigate to the "Order" dropdown and press Spacebar to expand it.
4. Use the Up and Down arrow keys to change the value.
<img width="401" alt="image" src="https://user-images.githubusercontent.com/11011881/186041217-f715cebe-6de4-4815-88a8-a8f8b199b2f0.png">

5. Navigate to 2nd page of the New Score dialog ("Additional score information")
6. Expand the Time Signature popup.
7. Tab to the denominator dropdown and press Spacebar to expand it.
8. Use the Up and Down arrow keys to change the dropdown value.
<img width="310" alt="image" src="https://user-images.githubusercontent.com/11011881/186040493-5eac23bb-d40c-44ae-83e7-b8a5560435b4.png">

**Expected behavior**
The screen reader should announce the dropdown value as it changes. Instead it just says "text" for each value.

**Platform information**
 - OS: macOS running VoiceOver
